backend/users/models.py

```diff
@@ -0,0 +1,13 @@
+from django.db import models
+from books.models import Book
+
+# Create your models here.
+
+class Student(models.Model):
+    Name = models.CharField(max_length = 40)
```

Comment:
Split this field into two, `first_name` and `last_name`, each with a max length of 20